tool/state-processor/src/vesting/mod.rs

```diff
@@ -14,11 +14,10 @@ impl<S> Processor<S> {
 		vestings.iter_mut().for_each(|(_, v)| v.adjust());
 
 		log::info!("set `AccountMigration::Vestings`");
-		{
-			let ik = item_key(b"AccountMigration", b"Vestings");
-
-			self.shell_state.insert_map(vestings, |h| format!("{ik}{h}"));
-		}
+		vestings.into_iter().for_each(|(k, v)| {
+			self.shell_state.inc_consumers_by(get_last_64(&k), 1);
```

Comment:
Why `inc_consumers` here?

I did not find any consumers update operation in the `polkadot-v0.9.33` branch. If I am missing something, please paste a link.